indexer/consensus_client.go

```diff
@@ -102,6 +108,12 @@ func (client *ConsensusClient) GetLastClientError() string {
 	return client.lastClientError.Error()
 }
 
+func (client *ConsensusClient) GetNodePeers() []*v1.Peer {
+	client.cacheMutex.RLock()
+	defer client.cacheMutex.RUnlock()
+	return client.peers
+}
```

Comment:
cacheMutex locking not required here :) 
but more important: 
maybe return a empty list here if a client is offline (not syncing) for some time?